utils/ratlib.py

```diff
@@ -41,6 +41,37 @@ class Status(Enum):
     """The rescue is open, but is marked inactive"""
 
 
+class Colors:
+    """
+    Contains mIRC-style color codes (the standard)
+    Reference: https://www.mirc.com/colors.html
+    """
+    WHITE = '\x030'
+    BLACK = '\x031'
+    BLUE = '\x032'
+    GREEN = '\x033'
+    RED = '\x034'
+    BROWN = '\x035'
+    PURPLE = '\x036'
+    ORANGE = '\x037'
+    YELLOW = '\x038'
+    LIGHT_GREEN = '\x039'
+    CYAN = '\x0310'
+    LIGHT_CYAN = '\x0311'
+    LIGHT_BLUE = '\x0312'
+    PINK = '\x0313'
+    GREY = '\x0314'
+    LIGHT_GREY = '\x0315'
+
+    # codes for formatting
+    FORMAT_BOLD = '\x02'
+    FORMAT_UNDERLINE = '\x1F'
+    FORMAT_ITALIC = '\x1D'
+    FORMAT_COLOR = '\x03'
```

Comment:
if you have the `FORMAT_COLOR` as a member of the enum, you do not also need to include it in the color enumerations.

I would suggest making `WHITE` thru `LIGHT_GREY` be ints rather than strings. would simplyify logic below.

Reply:
One more thing, i would suggest moving the bold/underline/italic/color/reverse/reset format keys to their own enum, as they are not technically colors